src/plotting.jl

```diff
@@ -724,6 +724,7 @@ pzmap
         @warn("pzmap currently only supports SISO systems. Only transfer function from u₁ to y₁ will be shown")
     end
     seriestype := :scatter
+    framestyle := :zerolines
```

Comment:
```suggestion
    framestyle --> :zerolines
```
This must use the arrow Syntax to make it a default rather than to force it.

Reply:
Nice, good to know. So you fixed it now, and the PR is updated with your fix? Or do I have to do anything about it?

I gotta admit, I don't understand these plot-recipes with macros. I guess it is for defining defaults, forcing, etc in a package, whereas in a script one would just pass it all to the plot-call? A short description or pointer to documentation on why one uses things like macros, and ```-->```/```:=```  in the recipes for this package instead of just ```=```, would be nice. I currently don't get it at all...